azerothcore/azerothcore-wotlk
# fix(DB/Creature): Incorrect jewelcrafting trainer

Shattrath Jewelcrafting trainer should not be able to learn grand master jewelcrafter. 
The NPC name is Hamanar(entry:19063) ,location is aldor area in shattrath.



## Changes Proposed:
-  Remove the grand master level training for this NPC.

## How to Test the Changes:


1. Find the NPC and confirm the bug exists (NPC provides grand master level training) 

> .go xyz -1654.68 5664.69 132.774 530

2. Delete the grand master level training of this NPC from database

> DELETE FROM `npc_trainer` WHERE `ID` = '19063' AND `SpellID` = '-201026';

3. Check the NPC again to make sure the problem is resolved.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
